core-ui/src/components/HelmReleases/HelmReleasesList.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import LuigiClient from '@luigi-project/client';
+import { useTranslation, Trans } from 'react-i18next';
+import {
+  useMicrofrontendContext,
+  useGetList,
+  PageHeader,
+  Link as ExternalLink,
+  Labels,
+  GenericList,
+} from 'react-shared';
+import { Link } from 'fundamental-react';
+import { decodeHelmRelease } from './decodeHelmRelease';
+import { findRecentRelease } from './findRecentRelease';
+
+const groupBy = (arr, fn) =>
+  arr.reduce((acc, curr) => {
+    if (!acc[fn(curr)]) {
+      acc[fn(curr)] = [];
+    }
+    acc[fn(curr)].push(curr);
+    return acc;
+  }, {});
+
+export function HelmReleasesList() {
+  const { t, i18n } = useTranslation();
+  const { namespaceId: namespace } = useMicrofrontendContext();
+
+  const { data, loading, error } = useGetList(
+    s => s.type === 'helm.sh/release.v1',
+  )(`/api/v1/namespaces/${namespace}/secrets`);
+
+  const headerRenderer = () => [
+    t('common.headers.name'),
+    t('common.headers.labels'),
+    t('helm-releases.headers.chart'),
+    t('helm-releases.headers.revision'),
```

Comment:
on the list, we have for example `revision` - 11, and `version` - 0.0.1, and on the details, we show `version` as 11. That should be `revision` I believe.